### PR TITLE
docs(contribution): pr template

### LINKED ISSUE #33: Add issue and pull request templates
We should add issues and pull request templates for better workflow.

### LINKED ISSUE #67: [FEATURE] PR Template
### Detailed description

Add a PR Template from LinkFree repo

### Context

-

### Possible implementation

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by suitupstan**
fixes #33 

fixes #67 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/EventCalendar/pull/74"></a>

### CODE REVIEW

**Review by shreyaansjain06 — APPROVED** [review]
LGTM

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+<!-- Remove this section if not applicable -->
+
+<!-- Example: Closes #31 -->
+
+## Changes proposed
+
+<!-- Here comes all the changes proposed through this PR -->
+
+<!-- Check all the boxes which are applicable to check the box correct follow the following conventions -->
```
Comment (EmmaDawsonDev) [inline .github/PULL_REQUEST_TEMPLATE.md]: I don't understand what this line means - can you reword it so it's easier to understand?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+- [ ] My code follows the code style of this project.
+- [ ] My change requires a change to the documentation.
+- [ ] I have updated the documentation accordingly.
+- [ ] All new and existing tests passed.
+- [ ] This PR does not contain plagiarized content.
+- [ ] The title of my pull request is a short description of the requested changes.
+
+## Screenshots
+
+<!-- Add all the screenshots which support your changes -->
```
Comment (EmmaDawsonDev) [inline .github/PULL_REQUEST_TEMPLATE.md]: Remove 'all the' so it reads "Add screenshots which support your changes"

**Review by EmmaDawsonDev — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (suitupstan) [reply .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
<!-- Add screenshots which support your changes -->
```

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (suitupstan) [reply .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
<!-- To check a box replace `[ ]` to `[x]` -->
```

**Review by starlightknown — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+## Fixes Issue
+
+<!-- Remove this section if not applicable -->
+
+<!-- Example: Closes #31 -->
+
+## Changes proposed
+
+<!-- Here comes all the changes proposed through this PR -->
```
Comment (EmmaDawsonDev) [inline .github/PULL_REQUEST_TEMPLATE.md]: How about ‘List all the proposed changes’ so that this is the same as the linkfree template.

Comment (suitupstan) [reply .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
<!-- List all the proposed changes -->
```

**Review by EmmaDawsonDev — APPROVED** [review]
LGTM!

**Pull request merged**